Syntax checker stopped working for Vue files

When saving a "vue" file, the following error appears in the console:
```
Error detected while processing function 77[1]..10:
line   18:
E700: Unknown function: SyntaxCheckers_javascript_eslint_GetLocList
Error detected while processing function <SNR>34_BufWritePostHook[3]..<SNR>34_UpdateErrors[13]..<SNR>34_CacheErrors[34]..16[1]..15:
line    3:
E15: Invalid expression: self._locListFunc()
Error detected while processing function <SNR>34_BufWritePostHook[3]..<SNR>34_UpdateErrors[13]..<SNR>34_CacheErrors:
line   34:
E170: Missing :endfor
Error detected while processing function <SNR>34_BufWritePostHook[3]..<SNR>34_UpdateErrors:
line   13:
E171: Missing :endif
```

### Vue file
```
<template>
<div class="monitor">
	<div>Monitor</div>
	<monitor />
</div>
</template>

<script>

import Monitor from './components/Monitor.vue'

export default {
	components: { Monitor },
	data () {
		return {
			msg: 'Hello from vue-loader!'
		}
	}
}
</script>

<style>
.message {
	color: blue;
}
</style>
```

### NPM eslint
```
± % npm ls -g | grep eslint
├─┬ eslint@4.18.2
│ ├─┬ eslint-scope@3.7.1
│ ├── eslint-visitor-keys@1.0.0
├── eslint-config-standard@11.0.0
├─┬ eslint-plugin-html@4.0.2
├─┬ eslint-plugin-import@2.9.0
│ ├─┬ eslint-import-resolver-node@0.3.2
│ ├─┬ eslint-module-utils@2.1.1
├─┬ eslint-plugin-node@6.0.1
├── eslint-plugin-promise@3.7.0
├── eslint-plugin-standard@3.0.1
├─┬ eslint-plugin-vue@4.3.0
│ └─┬ vue-eslint-parser@2.0.3
│   ├─┬ eslint-scope@3.7.1
│   ├── eslint-visitor-keys@1.0.0
```

### Variables
- g:loaded_syntastic_vue_eslint_checker is 1
- g:loaded_syntastic_javascript_eslint_checker is not defined

Is "g:loaded_syntastic_javascript_eslint_checker" supposed to be 1 so that vim-vue can redirect to it ?

From what I see the missing function `SyntaxCheckers_javascript_eslint_GetLocList` is defined in "Syntastic/syntax_checkers/javascript/eslint.vim" and likely not loaded because the above variable is not set.

Is vim even supposed to go there or should it call a different function ?

How to debug this ?

replacing "jshint" with
```
let g:syntastic_javascript_checkers = ['eslint']
```

doesn't help either.

When running eslint on the file from the CLI:
```
± % eslint src/App.vue 

src/App.vue
  8:1  error  Parsing error: Adjacent JSX elements must be wrapped in an enclosing tag

✖ 1 problem (1 error, 0 warnings)
```


If I open a regular JS file then the variable `g:loaded_syntastic_javascript_eslint_checker` is defined and the checker works.

So maybe the vue plugin is doing something that prevents the regular eslint checker to enable itself ?

--- a/syntax_checkers/vue/eslint.vim
+++ b/syntax_checkers/vue/eslint.vim
@@ -6,6 +6,8 @@ endif
 
 let g:loaded_syntastic_vue_eslint_checker = 1
 
+runtime! syntax_checkers/javascript/eslint.vim
+
 call g:SyntasticRegistry.CreateAndRegisterChecker({
       \   'filetype': 'vue',
       \   'name': 'eslint',

